src/server/tiered_storage_test.cc

```diff
@@ -273,14 +273,22 @@ TEST_F(TieredStorageTest, FlushAll) {
     }
   });
 
-  ExpectConditionWithinTimeout([&] { return GetMetrics().events.hits > 2; });
+  Metrics metrics;
+  ExpectConditionWithinTimeout([&] {
+    metrics = GetMetrics();
+    return metrics.events.hits > 2;
+  });
+  LOG(INFO) << FormatMetrics(metrics);
+
   Run({"FLUSHALL"});
 
   done = true;
-  util::ThisFiber::SleepFor(50ms);
+  util::ThisFiber::SleepFor(100ms);
```

Comment:
Why did you increase it ? Is this correlated with the test failure or ?

Reply:
maybe, not sure.